README.md

```diff
@@ -80,7 +80,7 @@ func logEvent(_ event: Event)
 
 IAM provides three pre-defined event types and a custom event type:
 
-1.  `AppStartEvent` - This event should be logged when the application is considered started by the host app. E.G AppDelegate's didFinishLaunchingWithOptions.
+1.  `AppStartEvent` - This event should be logged when the application is considered started by the host app. E.G AppDelegate's didFinishLaunchingWithOptions. It is persistent, meaning, once it's logged it will always satisfy corresponding trigger in a campaign. All subsequent logs of this event are ignored. Campaigns that require only AppStartEvent are shown once per app session.
```

Comment:
Should we define app session or is it too obvious?

Reply:
I thought about that. Maybe it's too vague. I'll try to be more specific